randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -1713,18 +1713,28 @@ Extra - asset_id: collision_camera_019
       Screw Attack
   > Event - Grapple Block (grapplepulloff1x2_000)
       All of the following:
-          Morph Ball
+          Grapple Beam and Morph Ball
           Any of the following:
               Cross Bomb
               All of the following:
                   # Bomb & Power Bombs, to desync the tunnel bomb blocks from the ledge bomb blocks
                   Bomb and Power Bombs
+              All of the following:
+                  # Power bombs (video: https://twitter.com/i/status/1545111396760690692 )
+                  Power Bombs ≥ 2
+              All of the following:
+                  # Bombs + flash shift (video: https://www.youtube.com/watch?v=NdVesJ6F_6A )
+                  Bomb and Flash Shift
   > Tile Group (WEIGHT)
       Morph Ball
   > Tunnel to Energy Recharge Station Secret
       All of the following:
           Morph Ball and After Ferenia - grapplepulloff1x2_000
-          Cross Bomb or Power Bombs
+          Any of the following:
+              Cross Bomb or Power Bombs ≥ 3
```

Comment:
Change this back to 1 Power Bomb, or just "Lay Power Bomb". You don't need to cover for getting back out of Energy Recharge Station Secret. I recommend putting "Lay Power Bomb" for all cases where a PB can be required as it covers both the Main item and at least one in inventory. If more are needed, you can list Power Bombs with it.

Reply:
Does the randomizer know how many powerbombs are needed if I change it back into 1 PB needed? If so, the room Energy Recharge Station Secret should be changed a little bit because one PB is sufficient to grab the pickup.

Reply:
Yes, the randomizer will know how many you need. If you are going to add the Secret room to the PR, I suggest removing the "Tile Group (BOMB WEIGHT)" node as that one is essentially useless, then you'll need to mark going to "Corner" to include Power Bombs. And then in the Gate room you need to add Power Bombs on "Tunnel to Energy Recharge Station Secret" to "Tile Group (WEIGHT)"